Docs 7695 pm 2.3 troubleshooting enhancements mc (#82)
mulesoft/docs-partner-manager: modules/ROOT/pages/ts-deploy-initialize-fail.adoc

> = Deployment Initialization failed

Reviewer: verbatim, I assume

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

== Cause

> Temporary System or connectivity issues between the Partner Manager control plane and the deployment server or (in case of CloudHub deployment) Anypoint MQ. 

Reviewer: ```suggestion
Your environment is experiencing temporary system or connectivity issues between the Partner Manager control plane and the deployment server or, in CloudHub deployment, Anypoint MQ. 
```

> == Solution

Reviewer: ```suggestion
== Solutions
```

> * Verify that your deployment server group *B2BServerGroup* is in the `Running` state.

Reviewer: Is this correct? I'm looking for a way to make these options parallel:
```suggestion
* For CloudHub deployments, verify that your deployment server group *B2BServerGroup* is in the `Running` state.
```

> For On-Premises, try redeploying the message flow.

Reviewer: ```suggestion
* For on-premises deployments, attempt to redeploy the message flow, wait 10 minutes, and try again.
```

* Try redeploying the message flow after 10 minutes.

> 

Reviewer: see above

> If the error persists, create a support ticket with MuleSoft along with the Reference number in the deployment error.

Reviewer: ```suggestion
If the error persists, create a MuleSoft support ticket, including the error reference number.
```

Reviewer: BREAK HERE
